src/helpers/sentry.ts

```diff
@@ -3,6 +3,10 @@ import type { Express } from 'express';
 import { rpcError } from './utils';
 
 export function initLogger(app: Express) {
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
let's add similar lines to `fallbackLogger` and `capture` with respect to their purposes

Reply:
Did not test yet, but isn’t it already taken care by sentry itself ?